tests/test_views.py

```diff
@@ -1,38 +0,0 @@
-from cms.test_utils.testcases import CMSTestCase
-from cms.utils.urlutils import admin_reverse
-
-from .utils.factories import SnippetWithVersionFactory
-
-
-class PreviewViewTestCase(CMSTestCase):
```

Comment:
We should still have a test that the form renders and the fields are readonly or else we now have a whole new set of functionality without any tests. This is especially important as we are re-implementing django components, if django 4 changes anything on the changeform / changeform templates we need the test suite to tell us. 

I think this can be achieved in one test that renders the form and checks the fields are readonly.